crates/ruma-events/src/enums.rs

```diff
@@ -414,7 +421,34 @@ impl AnyMessageLikeEventContent {
                 Some(encrypted::Relation::Reference(relates_to.clone()))
             }
             #[cfg(feature = "unstable-msc3381")]
-            Self::PollStart(_) | Self::UnstablePollStart(_) => None,
+            Self::UnstablePollStart(UnstablePollStartEventContent::New(content)) => {
+                match &content.relates_to {
+                    Some(RelationWithoutReplacement::Thread(thread)) => {
+                        Some(encrypted::Relation::Thread(thread.clone()))
+                    }
+                    Some(RelationWithoutReplacement::Reply { in_reply_to }) => {
+                        Some(encrypted::Relation::Reply { in_reply_to: in_reply_to.clone() })
+                    }
+                    Some(RelationWithoutReplacement::_Custom(_)) | None => None,
+                }
+            }
+            #[cfg(feature = "unstable-msc3381")]
+            Self::UnstablePollStart(UnstablePollStartEventContent::Replacement(content)) => {
+                Some(encrypted::Relation::Replacement(Replacement::new(
+                    content.relates_to.event_id.clone(),
+                )))
+            }
+            #[cfg(feature = "unstable-msc3381")]
+            Self::PollStart(PollStartEventContent { relates_to, .. }) => match relates_to {
+                Some(Relation::Thread(thread)) => Some(encrypted::Relation::Thread(thread.clone())),
+                Some(Relation::Reply { in_reply_to }) => {
+                    Some(encrypted::Relation::Reply { in_reply_to: in_reply_to.clone() })
+                }
+                Some(Relation::Replacement(replacement)) => Some(encrypted::Relation::Replacement(
+                    Replacement::new(replacement.event_id.clone()),
+                )),
+                Some(Relation::_Custom(_)) | None => None,
+            },
```

Comment:
This should be like other branches that use `room::message::Relation`:

```suggestion
            Self::PollStart(ev) => {
                ev.relates_to.clone().map(Into::into)
            },
```

Reply:
Thanks, I didn't realise this was possible, there are so many different types of relations here 😅 .